vector/src/main/java/im/vector/app/features/roomdirectory/createroom/CreateRoomController.kt

```diff
@@ -165,7 +165,11 @@ class CreateRoomController @Inject constructor(
                             host.stringProvider.getString(R.string.create_room_encryption_description)
                         }
                 )
-                switchChecked(viewState.isEncrypted)
+                if (viewState.isEncrypted != null) {
```

Comment:
NIT: Something I've noticed through the codebase is we tend to favour inverted if conditions, it might just be me but I find positive conditions much easier on the brain 

```kotlin
if (viewState.isEncrypted == null) {
  switchChecked(viewState.defaultEncrypted[viewState.roomJoinRules] ?: false)
} else {
  switchChecked(viewState.isEncrypted)
}
``` 

and whilst inverting the if ^^^ I also noticed we could simplify the checked state by chaining the fallbacks

```kotlin
val isSwitchChecked = viewState.isEncrypted ?: viewState.defaultEncrypted[viewState.roomJoinRules] ?: false
```

Reply:
Generally I agree to avoid `if(!cond) {} else {}`, but sometimes it also nice to have the most common case first. I'm not saying this is the case here, just wanted to add my two pence :)

Reply:
Yes I explicitly used the if/else like that thinking it will ease reading of the code (otherwise I would have gone for the chaining).
Looks like I failed  :D.